mininet/simpletest.py

```diff
@@ -1,16 +1,21 @@
 import json
 import os
 import subprocess
+import atexit
 
-import mininet.log as log
 from mininet.cli import CLI
 from mininet.net import Mininet
+from mininet.log import setLogLevel, info
+from mininet.util import dumpNetConnections
 
 from nodes import Floodlight
 from topologies.simplenobgptopo import SimpleNoBGPTopo
 
+from os import path
+from os import makedirs
+
 HOME_FOLDER = os.getenv('HOME')
-LOG_PATH = HOME_FOLDER + '/TARN/logs/'
+LOG_PATH = HOME_FOLDER + '/TARNProject/TARN/logs/'
```

Comment:
We're probably going to want to keep the path set to `$HOME/TARN/logs/`, just because `TARNProject` is a folder that's unique to your environment